src/components/cms/BotBuilder/BotBuilderPages/DesignViews/UIView.js

```diff
@@ -343,14 +344,11 @@ class UIView extends Component {
               ) ? (
                 <div style={{ display: 'flex', flexDirection: 'row' }}>
                   <div className="color-picker-wrap">
-                    <span
-                      className="color-box"
+                    <ColorBox
+                      backgroundColor={this.props.design[component.component]}
                       onClick={() => this.handleClickColorBox(component.id)}
-                      style={{
-                        backgroundColor: this.props.design[component.component],
-                      }}
                     />
-                    <div className="colorPicker">
+                    <ColorPickerContainer>
```

Comment:
Move the entire color picker to the shared file.

Reply:
@akshatnitd fixed.